schema/Retail/RET11/on_search.ts

```diff
@@ -515,6 +553,8 @@ export const FnBonSearchSchema = {
                         },
                         '@ondc/org/time_to_ship': {
                           type: 'string',
+                          pattern: '^PT(?:(?:60|[1-5]?[0-9]|60)M|1H)$',
```

Comment:
change it into duration format

Reply:
That was for RET11, that it cannot be greater than 1 hour